src/code/z_kankyo.c

```diff
@@ -183,7 +188,7 @@ f32 sSandstormLerpScale = 0.0f;
 
 u8 gCustomLensFlareOn;
 Vec3f gCustomLensFlarePos;
-s16 gLensFlareUnused;
+s16 sLensFlareUnused;
 s16 gLensFlareScale;
 f32 gLensFlareColorIntensity;
 s16 gLensFlareGlareStrength;
```

Comment:
this change doesnt make too much sense to me, cause the other lens flare variables are meant to be accessed globally to change its properties. I would assume that this variable followed that same pattern, when/if it actually meant something. But since its unused I guess I dont care too much

Reply:
I think the BSS ordering suggests that this variable was not declared in a header (or at least, not in the same place as the other variables surrounding it here) since only `sLensFlareUnused` is reordered in retail